cmd/xl-v1-list-objects-heal.go

```diff
@@ -27,3 +27,8 @@ func (xl xlObjects) ListBucketsHeal(ctx context.Context) ([]BucketInfo, error) {
 func (xl xlObjects) HealObjects(ctx context.Context, bucket, prefix string, fn healObjectFn) (e error) {
 	return nil
 }
+
+// this is not implemented/needed anymore, look for xl-sets.Walk()
+func (xl xlObjects) Walk(ctx context.Context, bucket, prefix string, doneCh chan struct{}) <-chan ObjectInfo {
+	return nil
```

Comment:
```suggestion
	return NotImplemented{}
```

Reply:
This is not correct. Walk doesn't return any error just returns nil channel i.e no entries.